src/mono/wasm/runtime/dotnet.d.ts

```diff
@@ -153,7 +156,70 @@ type MonoConfig = {
      * query string to be used for asset loading
      */
     assetUniqueQuery?: string;
+    /**
+     * Gets the application culture. This is a name specified in the BCP 47 format. See https://tools.ietf.org/html/bcp47
+     */
+    applicationCulture?: string;
+    /**
+     * Overrides the built-in boot resource loading mechanism so that boot resources can be fetched
+     * from a custom source, such as an external CDN.
+     * @param type The type of the resource to be loaded.
+     * @param name The name of the resource to be loaded.
+     * @param defaultUri The URI from which the framework would fetch the resource by default. The URI may be relative or absolute.
+     * @param integrity The integrity string representing the expected content in the response.
+     * @returns A URI string or a Response promise to override the loading process, or null/undefined to allow the default loading behavior.
+     */
+    loadBootResource?: LoadBootResourceCallback;
+    /**
+     * exports from library es6 modules
+     *
+     * nuget packages can contain wwwroot/*.lib.module.js which are treated as es6 modules
+     * runtime calls 'onRuntimeConfigLoaded(config: MonoConfig)' and 'onRuntimeReady(api: RuntimeAPI)'
+     * blazor calls 'beforeStart' and 'afterStarted'
+     */
+    libraryInitializers?: any[];
+    /**
+     * definition of assets to load along with the runtime.
+     */
+    resources?: ResourceGroups$1;
```

Comment:
`ResourceGroups$1` is ugly, maybe you need to be more specific that you are exporting this type on API level.

Reply:
Should we drop `assets` ? Or is that only after Blazor changes ?

Reply:
We will drop `assets` in favor of `resources` (not in this PR). I'll look into the ugly name. Thanks, I missed it!

Reply:
It was caused by exporting the `ResourceGroups` in both `MonoConfig` and `BootJsonData`, but `BootJsonData` is not needed to be exported anymore